querybook/webapp/components/DataDocStatementExecution/DataDocStatementExecution.tsx

```diff
@@ -148,13 +151,36 @@ export const DataDocStatementExecution: React.FC<IProps> = ({
                     />
                 </div>
             );
+            const estimatedTimeDOM = (() => {
+                if (percentComplete === 0 || percentComplete === 100) {
+                    return null;
+                }
+
+                const secondsPassedSinceStart =
+                    new Date().getTime() / 1000 - createdAt;
+                const secondsRemainToComplete =
+                    secondsPassedSinceStart / (percentComplete / 100) -
+                    secondsPassedSinceStart;
+
+                return (
+                    <div>
+                        Estimated to be completed in{' '}
+                        {formatDuration(
+                            moment.duration(secondsRemainToComplete, 'seconds')
+                        )}
+                        .
+                    </div>
+                );
+            })();
+
             contentDOM = (
                 <div className="statement-execution-text-container">
                     <div className="statement-execution-text-title">
                         Status: {statusLabel}
                     </div>
                     {getMetaInfoDOM()}
                     {progressBar}
+                    {estimatedTimeDOM}
```

Comment:
how about display the estimation in the same line with progress bar, after the %, like
[>>>>.........] 25% (Estimated to be completed in 14s)